packages/core/src/lib/nav-dropdown/nav-dropdown.svelte

```diff
@@ -0,0 +1,137 @@
+<script lang="ts">
+import { clickOutside } from '$lib/click-outside';
+import { Icon } from '$lib/icon';
+import { goto } from '$app/navigation';
```

Comment:
Is goto from app/navigation the right thing to use to navigate to the selected href?

Reply:
Removed, ty!